src/sendCommercialMetrics.ts

```diff
@@ -107,15 +123,49 @@ function sendMetrics(isDev: boolean) {
  * A method to asynchronously send metrics after initialization.
  * @param init.isDev - used to determine whether to use CODE or PROD endpoints.
  */
-export function bypassCommercialMetricsSampling(isDev: boolean): void {
+export function bypassCommercialMetricsSampling(): void {
 	if (!initialised) {
 		console.warn('initCommercialMetrics not yet initialised');
 		return;
 	}
 
-	sendMetrics(isDev);
+	sendMetrics();
 }
 
+export function gatherMetricsOnPageUnload(): void {
```

Comment:
Is this only exported for testing? If so, it would be preferable to make it part of the `_` export.

Reply:
Actually, I don't think it needs exporting at all.